### ISSUE TITLE
Visual Script with Int16 Parameter causes issues

### ISSUE THREAD

**Issue by stefnotch** [FlaxEngine/FlaxEngine#497]
**Issue description:**
When you add a parameter of the type `Int16` to a visual script and then attach the visual script to an actor, Flax will loudly complain

![image](https://user-images.githubusercontent.com/10220080/116739267-c1e3a200-a9f3-11eb-9287-e1a10a4fd311.png)

**Steps to reproduce:**
1. Create a visual script
2. Add a `Int16` parameter 
3. Save the visual script
4. Attach the visual script to an actor (in the editor)
5. As soon as you attach it, the output log will start spamming this in the output log

```
[Error] Invalid type to box Structure System.Int16
[ 00:00:13.653 ]: [Warning] Failed to setup values and UI for item IndexFace
[ 00:00:13.653 ]: [Warning] Object reference not set to an instance of an object.
[ 00:00:13.656 ]: [Warning]   at FlaxEditor.CustomEditors.Editors.ShortEditor.GetValue (System.Object value) [0x00000] in C:\Flax\FlaxEngine\Source\Editor\CustomEditors\Editors\IntegerEditor.cs:216 
  at FlaxEditor.CustomEditors.Editors.SignedIntegerValueEditor.Refresh () [0x0000e] in C:\Flax\FlaxEngine\Source\Editor\CustomEditors\Editors\IntegerEditor.cs:148 
  at FlaxEditor.CustomEditors.CustomEditor.Initialize (FlaxEditor.CustomEditors.CustomEditorPresenter presenter, FlaxEditor.CustomEditors.LayoutElementsContainer layout, FlaxEditor.CustomEditors.ValueContainer values) [0x0002f] in C:\Flax\FlaxEngine\Source\Editor\CustomEditors\CustomEditor.cs:125 
  at FlaxEditor.CustomEditors.LayoutElementsContain...
  ```

**Flax version:**
Flax 1.2

**Comment by LCRW** [comment]
Its in master also, looks like a threading issue maybe., Haven't been able to lock down error in debug.

**Comment by LCRW** [comment]
Awesome work

**Comment by jb-perrier** [comment]
Fixed in #515 .
